frontend/src/pages/MessagingQueues/MessagingQueues.styles.scss

```diff
@@ -310,3 +295,119 @@
 		}
 	}
 }
+
+.lightMode {
+	.messaging-queue-container {
+		.messaging-breadcrumb {
+			color: var(--bg-ink-400);
+			border-bottom: 1px solid var(--bg-vanilla-300);
+		}
+		.messaging-header {
+			color: var(--bg-ink-400);
+			border-bottom: 1px solid var(--bg-vanilla-300);
+
+			.header-config {
+				.messaging-queue-options {
+					.ant-select-selector {
+						border: 1px solid var(--bg-vanilla-300);
+						background: var(--bg-vanilla-100);
+					}
+				}
+			}
+		}
+
+		.messaging-queue-main-graph {
+			.config-options {
+				.config-select-option {
+					.ant-select-selector {
+						border: 1px solid var(--bg-vanilla-300);
+						background: var(--bg-vanilla-100);
+					}
+				}
+			}
+
+			border-bottom: 1px solid var(--bg-vanilla-300);
+		}
+
+		.messaging-queue-details {
+			.mq-details-options {
+				.ant-radio-button-wrapper {
+					border-color: var(--bg-vanilla-300);
+					color: var(--bg-slate-200);
+				}
+				.ant-radio-button-wrapper-checked {
+					color: var(--bg-slate-200);
+					background: var(--bg-vanilla-300);
+				}
+				.ant-radio-button-wrapper-disabled {
+					background: var(--bg-vanilla-100);
+					color: var(--bg-vanilla-400);
+				}
+			}
+		}
+	}
+
+	.messaging-overview {
+		.overview-text {
+			color: var(--bg-slate-200);
+		}
+
+		.overview-subtext {
+			color: var(--bg-slate-300);
+		}
+
+		.overview-doc-area {
+			.overview-info-card {
+				border: 1px solid var(--bg-vanilla-300);
+
+				.card-title {
+					color: var(--bg-slate-200);
+				}
+
+				.card-info-text {
+					color: var(--bg-slate-300);
+				}
+
+				.button-grp {
+					.ant-btn-default {
+						background-color: var(--bg-vanilla-100);
+					}
+				}
+			}
+		}
+
+		.summary-section {
+			.summary-card {
+				border: 1px solid var(--bg-vanilla-300);
+				background: var(--bg-vanilla-100);
+
+				.summary-title {
+					> p {
+						color: var(--bg-slate-200);
+					}
+
+					.time-value {
+						> p {
+							color: var(--bg-slate-200);
+						}
+					}
+				}
+			}
+		}
+
+		.coming-soon-card {
+			background: var(--bg-vanilla-200) !important;
```

Comment:
should we remove the `!important`? as I checked, we get the same result if we remove the !important from here this snippet above in the file:

```
.coming-soon-card {
  background: var(--bg-ink-500) !important;
  border-left: none !important;
}

```

Reply:
I checked but I do require the `!important` here